client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions-i2/show-hide-input/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button } from '@automattic/components';
+import { ClipboardButton } from '@wordpress/components';
+import { Icon, seen, unseen } from '@wordpress/icons';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { FC, useState, useMemo, useRef, useEffect } from 'react';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import './style.scss';
+
+interface Props {
+	value: string;
+	className?: string;
+}
+export const ShowHideInput: FC< Props > = ( { value, className } ) => {
```

Comment:
Do we have changes on this component? If you we can use the original.

Reply:
Yes we can, I just copy the whole instructions dir and this was there :). For now I will link to the original ones and then we can cleanup and move them outside of a step directory.

Reply:
https://github.com/Automattic/wp-calypso/issues/89899